docs/online-lfp.rst

```diff
@@ -0,0 +1,144 @@
+=======================================
+Online LFP Calculation Documentation
+=======================================
+
+Electrodes Input File
+---------------------
+
+Required Format
+~~~~~~~~~~~~~~~~
+
+To perform online LFP calculation, a weights file is required. The weights file should follow a specific format to ensure proper functioning. 
+More information about this file can be found in the `SONATA Simulation Specification <https://github.com/BlueBrain/sonata-extension/blob/master/source/sonata_tech.rst#format-of-the-electrodes_file>`_
+
+Generating the Electrodes File (WIP)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The electrodes file can be generated using specific steps and considerations. Follow the instructions below to generate the file:
+
+1. Produce a compartment report from a target including the cells that will be used for the LFP calculation.
+
+2. Run the file getPositions.py, using the bash script GetPositions.sh. This will create a folder containing pickle files listing the (x,y,z) position of each segment in each cell in the target.
```

Comment:
Where are these scripts for generating the electrodes file? Will they be stored in neurodamus or in other repo?

Reply:
Joseph has them in his public repo.
https://github.com/joseph-tharayil/create_lfp_weights_for_neurodamus

I told him to maybe create a bbp gitlab or bbp github for that but he doesnt have permissions. So i was planning to add the path to the repo once sorted out.